tests/5.0/teams_loop/test_target_teams_loop_defaultmap.c

```diff
@@ -0,0 +1,238 @@
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+
+#define N 1024
+
+/**
+  This is a basic test to demonstrate defaultmap(tofrom)
+  clause with target teams loop construct.
+*/
+int testDefaultMapToFrom() {
+  int errors = 0;
+  int device_data[N], host_data[N];
+  for (int i = 0; i < N; i++) {
+    device_data[i] = i;
+    host_data[i] = 0;
+  }
+#pragma omp target teams loop defaultmap(tofrom)
+  for (int i = 0; i < N; i++) {
+    host_data[i] = device_data[i];
```

Comment:
Should be  host_data[i] += device_data[I]; //to verify if host_data was copied correctly.

Reply:
Made changes as suggested...